### ISSUE TITLE
Let resolve API call accept a URL

### ISSUE THREAD

**Issue by jwheare** [RJ/playdar-core#6]
If we're resolving from an XSPF or podcast, there should be a way to seed playdar with any URLs included in the XSPF. This will then be returned in the results with the source as the domain, a perfect score, but maybe a lower pref than locally resolved tracks.

playdar.js can already send this in resolve calls:

http://github.com/jwheare/playdar.js/commit/760842b55bb07cdd40cb69878d6d67bd12f33a4b

**Comment by jwheare** [comment]
Regarding notes in http://github.com/RJ/playdar-core/commit/2c5c71a65db255739accea0aeb0a458c560110c6, I'm happy for the client to pass in the URL, metadata and source name (domain of the URL) as a result object, but the daemon should add score, sid, and preference values I think.

Bitrate, size and duration may not be available I guess but that's OK.

**Comment by RJ** [comment]
yep thats fine, we already autoadd score (soon) and sids etc to results from scripts, so that'll just work. How will the http call look?

**Comment by jwheare** [comment]
some sort of urlencoded JSON object I suppose.

**Comment by jwheare** [comment]
Checkit!
http://github.com/jwheare/playdar.js/issues/closed#issue/7

**Comment by lucasgonze** [comment]
This gives compatibility with the huge installed base of XSPF files that are basically just M3U, including most of the Musicplayer playlists.
